x-pack/plugins/siem/public/components/notes/helpers.tsx

```diff
@@ -54,9 +54,7 @@ export const NotesCount = pure<{
   noteIds: string[];
 }>(({ noteIds }) => (
   <EuiTitle size="s">
-    <TitleText>
-      {noteIds.length} Note{noteIds.length === 1 ? '' : 's'}
-    </TitleText>
+    <TitleText>{i18n.NOTE(noteIds.length)}</TitleText>
```

Comment:
Will this be fine when `noteIds.length === 0`? Looks like the [kibana i18n impl](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#pluralization) uses the [ICU Message syntax](http://userguide.icu-project.org/formatparse/messages), and 0 will be its own value?

Reply:
Nevermind, looks like it'll be replaced as `0 Notes`, which is 👍